lms/migrations/versions/1872d16c28a4_enable_the_youtube_enabled_setting_for_.py

```diff
@@ -0,0 +1,25 @@
+"""Enable the youtube.enabled setting for all application instances.
+
+Set the youtube.enabled setting to true for all application instances that
+currently have it explicitly set to false in the DB.
+
+Revision ID: 1872d16c28a4
+Revises: 106d94be7705
+"""
+import sqlalchemy as sa
+from alembic import op
+
+revision = "1872d16c28a4"
+down_revision = "106d94be7705"
+
+
+def upgrade() -> None:
+    op.get_bind().execute(
+        """UPDATE application_instances
```

Comment:
Tested this on MB, look like 109 AIs will be updated :+1: 

https://metabase.hypothes.is/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJzZWxlY3QgIHNldHRpbmdzIHx8IGpzb25iICd7XCJ5b3V0dWJlXCI6e1wiZW5hYmxlZFwiOiB0cnVlfX0nIGZyb20gYXBwbGljYXRpb25faW5zdGFuY2VzIFdIRVJFIHNldHRpbmdzIC0-ICd5b3V0dWJlJyAtPiAnZW5hYmxlZCcgPSAnZmFsc2UnOyIsInRlbXBsYXRlLXRhZ3MiOnt9fSwiZGF0YWJhc2UiOjd9LCJkaXNwbGF5IjoidGFibGUiLCJwYXJhbWV0ZXJzIjpbXSwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6eyJ0YWJsZS5jb2x1bW5fd2lkdGhzIjpbODk1XX19